doc/devel/development.md

```diff
@@ -0,0 +1,1946 @@
+| `subservice`                   | `subservice`                   |                                                                                                                                                   |
+| `resource`                     | `resource`                     |                                                                                                                                                   |
+| `apikey`                       | `apikey`                       |                                                                                                                                                   |
+| `timestamp`                    | `timestamp`                    |                                                                                                                                                   |
+| `entity_type`                  | `entity_type`                  |                                                                                                                                                   |
+| `trust`                        | `trust`                        |                                                                                                                                                   |
+| `cbHost`                       | `cbHost`                       |                                                                                                                                                   |
+| `lazy`                         | `lazy`                         |                                                                                                                                                   |
+| `commands`                     | `commands`                     |                                                                                                                                                   |
+| `attributes`                   | `attributes`                   |                                                                                                                                                   |
+| `static_attributes`            | `staticAttributes`             |                                                                                                                                                   |
+| `internal_attributes`          | `internalAttributes`           |                                                                                                                                                   |
+| `expressionLanguage`           | `expresionLanguage`            | _Removed_                                                                                                                                         |
```

Comment:
```suggestion
```

Given it its decreated functionality and acccoring to https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/deprecated.md

> Documentation on deprecated features is removed from the repository documentation.

Reply:
In this case, it is better to do in a subsequent PR. Noted here: https://github.com/telefonicaid/iotagent-node-lib/issues/1329#issue-1643605738
NTC